public/app/plugins/datasource/tempo/SearchTraceQLEditor/SearchField.tsx

```diff
@@ -112,13 +113,24 @@ const SearchField = ({
       operatorList = numberOperators;
   }
 
+  /**
+   * Add to a list of options the current template variables.
+   *
+   * @param options a list of options
+   * @returns the list of given options plus the template variables
+   */
+  const withTemplateVariableOptions = (options: SelectableValue[] | undefined) => {
+    const templateVariables = getTemplateSrv().getVariables();
+    return [...(options || []), ...templateVariables.map((v) => ({ label: v.name, value: `$${v.name}` }))];
```

Comment:
Would be great if the label included a `$` sign so users know it's a var. There is that sign in the PR description image but I guess that's because there was a dollar in the name itself!